app/controllers/api/v1/authentication_controller.rb

```diff
@@ -152,6 +152,15 @@ def current_user_profile
       render json: current_user, serializer: Api::V1::UserProfileSerializer
     end
 
+    api :POST, "/v1/auth/register_device", "Register a mobile device to receive notifications"
+    param :handle, String, desc: "The registration id for the push messaging service for the platform i.e. gcm registration id for android"
+    param :platform, String, desc: "The azure notification platform name, this should be `gcm` for android"
+    def register_device
+      authorize!(:register, :device)
+      render text: "Only 'gcm' platform is supported at this stage", status: 400 if params[:platform] != 'gcm'
```

Comment:
Line is too long. [110/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.